deluge/ui/web/js/deluge-all/preferences/InterfacePage.js

```diff
@@ -13,16 +13,22 @@ Ext.namespace('Deluge.preferences');
  * @class Deluge.preferences.Interface
  * @extends Ext.form.FormPanel
  */
-Deluge.preferences.Interface = Ext.extend(Ext.form.FormPanel, {
-
-    border: false,
-    title: _('Interface'),
-    layout: 'form',
+Deluge.preferences.Interface = Ext.extend(Ext.FormPanel, {
+    constructor: function(config) {
+        config = Ext.apply({
+            border: false,
+            title: _('Interface'),
+            layout: 'form',
+            autoHeight: true,
+            width: 320
+        }, config);
+        Deluge.preferences.Interface.superclass.constructor.call(this, config);
+    },
 
     initComponent: function() {
         Deluge.preferences.Interface.superclass.initComponent.call(this);
 
-        var om = this.optionsManager = new Deluge.OptionsManager();
+        var om = this.optionsManager = deluge.preferences.getOptionsManager();
```

Comment:
This causes an error when trying to save when not connected to a daemon. It should be possible to save the Interface page when not connected.